tools/telemetry-generator/package.json

```diff
@@ -37,11 +37,14 @@
 	"devDependencies": {
 		"@fluidframework/build-common": "^2.0.1",
 		"@fluidframework/eslint-config-fluid": "^5.2.0",
+		"@types/jest": "^29.5.13",
 		"@types/node": "^14.18.0",
 		"concurrently": "^8.2.1",
 		"eslint": "~8.55.0",
+		"jest": "^29.7.0",
```

Comment:
I'd suggest we stick to `mocha` for unit-testing like we do in most packages. We also have some nice infrastructure with `@fluid-internal/mocha-test-setup` and extending its `mocharc-common.js` file, which handles many of our scenarios ([example of how that's used](https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/.mocharc.cjs)).

Reply:
Okok. Let me covert them to mocha